__init__.py

```diff
@@ -21,6 +21,9 @@
  ***************************************************************************/
  This script initializes the plugin, making it known to QGIS.
 """
+import sys, os
+
+sys.path.append(os.path.join(os.path.realpath(__file__),os.pardir))
```

Comment:
Goed punt, regel was per ongeluk blijven staan. Imports werken ondertussen naar behoren, dus regel is in volgende pull request verwijderd.

Alle pep 8 checks moeten nog over de code heen.